ReactiveCocoa/Swift/Event.swift

```diff
@@ -49,7 +57,7 @@ public enum Event<T, E: ErrorType> {
 	public func map<U>(f: T -> U) -> Event<U, E> {
 		switch self {
 		case let .Next(value):
-			return Event<U, E>.Next(Box(f(value.unbox)))
+			return Event<U, E>.Next(Box(f(value.value)))
```

Comment:
I was under the impression that RAC was iOS8+, but I guess that’s my misunderstanding!